pkg/cli/cmd/uninstall/kubernetes/kubernetes.go

```diff
@@ -97,6 +102,14 @@ func (r *Runner) Run(ctx context.Context) error {
 		return err
 	}
 
+	if r.Purge {
+		r.Output.LogInfo("Deleting namespace...")
```

Comment:
This could be a bit explicit, "Deleting `radius-system` namespace...". @willtsai / @Reshrahim what do you think?

Reply:
I think that would be a good improvement.